MAUI/DataGrid/Grouping.md

```diff
@@ -0,0 +1,572 @@
+
+`GroupColumnDescription` object holds following two properties:
+
+ * ColumnName: Name of the grouped column.
+ * Converter: Get the `IValueConverter` as input that helps to apply the custom grouping.
+
+To apply column grouping, follow the code example:
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid x:Name="dataGrid"
+            ItemsSource="{Binding OrdersInfo}">
+
+    <syncfusion:SfDataGrid.GroupColumnDescriptions>
+        <syncfusion:GroupColumnDescription ColumnName="Name" />
+    </syncfusion:SfDataGrid.GroupColumnDescriptions>
+</syncfusion:SfDataGrid> 
+{% endhighlight %}
+{% highlight c# %}
+    dataGrid.GroupColumnDescriptions.Add (new GroupColumnDescription () {
+    ColumnName = "Name",
+}); 
+{% endhighlight %}
+{% endtabs %}
+
+The following screenshot shows the output rendered when grouping is applied:
+
+![DataGrid with grouping](Images\Grouping\SingleGrouping.png)
+
+## MultiGrouping
+
+The SfDataGrid also allows to group the data against one or more columns using the [SfDataGrid.GroupingMode]() property. When `GroupingMode` is set as `GroupingMode.Multiple`, the data is organized into hierarchical tree structure based on identical values of that column. MultiGrouping feature works similarly as MultiSorting feature. Initially the data is grouped according to the first column added in the `GroupColumnDescriptions` collection. When more columns are added to the `GroupColumnDescriptions`, the newly added column will be grouped in consideration to the previous group(s). This results in a tree like hierarchy. Refer to the following code snippet to enable `MultiGrouping`:
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid  x:Name="dataGrid"
+                        ItemsSource="{Binding Orders}"
+                        GroupingMode = "Multiple"/>
+{% endhighlight %}
+
+{% highlight c# %}
+
+this.dataGrid.GroupingMode = GroupingMode.Multiple;
+
+{% endhighlight %}
+{% endtabs %}
+
+The following screenshot shows the output rendered when above code is executed:
+![DataGrid with multi-column grouping](Images\Grouping\MultipleGrouping.png)
+
+## Indent column customizations
+
+Indent columns are the columns present to the left of the `CaptionSummaryRows` when `GroupingMode` is set as multiple. The number of indent cells in each `CaptionSummaryRow` will be determined by the level of that `Group`. For example, the first group will have only one indent cell and the next immediate group will have an extra indent cell. It keeps on adding by one for each lower level groups to maintain the tree structure. Each data row will have indent cells count equal to the level of the last sub group in view. The following customizations can be done for indent cells:
+
+### Customize indent column width
+
+By default, the width of the indent column is 20. To customize the width of indent column, use the [IndentColumnWidth]() property as follows: 
```

Comment:
The width of indent column can be customized by the [IndentColumnWidth]() property. The default width of the indent column is 35.

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/f69e97f7-33e1-4e4f-92ac-80cc0bf271c4)